src/styles/blocks/page.scss

```diff
@@ -0,0 +1,8 @@
+@use '../_variables/variables' as *;
+
+.page {
+  font-family: $PrimaryFont;
+  color: $ColorMainAccent;
+  font-weight: $FontRegular;
```

Comment:
The variable `$FontRegular` should be in lowercase according to the code style rules. Please rename it to `$font-regular`.